Bio/Align/fasta_m8.py

```diff
@@ -0,0 +1,238 @@
+# Copyright 2008-2016 by Peter Cock.  All rights reserved.
```

Comment:
This should be copyright you, but can acknowledge based on code by me. Or list both names as copyright holders as you prefer.

Reply:
Replaced by my name, as the code is new.